dev-requirements.txt

```diff
@@ -4,10 +4,10 @@
 
 # chartpress is relevant to build and push helm-charts/images/hub/Dockerfile and
 # update basehub's default values to reference the new image.
-chartpress==2.1.0
+chartpress
```

Comment:
The PR states pinning to major version but that's not happening here and below for requests. 

Can you add major pinning for those two

Reply:
Makes sense, done!